packages/renderer/noteStyle.ts

```diff
@@ -434,6 +440,15 @@ export default function(theme: any, options: Options = null) {
 			overflow-x: auto;
 		}
 
+		#rendered-md {
+			${overflowHidden}
+		}
+
+		.joplin-table-div{
+			overflow-x: overlay;
```

Comment:
```suggestion
			overflow-x: auto;
```

It looks like [`overflow: overlay;`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#auto) is [a legacy alias for `overflow: auto`](https://drafts.csswg.org/css-overflow/#valdef-overflow-overlay) — consider using `overflow-x: auto` here instead.

Reply:
Fixed